modules/aggregation-layers/src/screen-grid-layer/screen-grid-cell-layer.js

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2015 - 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import GL from '@luma.gl/constants';
+import {Model, Geometry, FEATURES, hasFeatures} from '@luma.gl/core';
+import {Layer, log} from '@deck.gl/core';
+import {defaultColorRange, colorRangeToFlatArray} from '../utils/color-utils';
+import vs from './screen-grid-layer-vertex.glsl';
+import fs from './screen-grid-layer-fragment.glsl';
+
+const DEFAULT_MINCOLOR = [0, 0, 0, 0];
+const DEFAULT_MAXCOLOR = [0, 255, 0, 255];
+const COLOR_PROPS = [`minColor`, `maxColor`, `colorRange`, `colorDomain`];
+
+const defaultProps = {
+  cellSizePixels: {value: 100, min: 1},
+  cellMarginPixels: {value: 2, min: 0, max: 5},
+
+  colorDomain: null,
+  colorRange: defaultColorRange
+};
+
+export default class ScreenGridCellLayer extends Layer {
```

Comment:
Add standard tests for this layer.

Reply:
Added tests, couldn't use auto generated tests, as they contain `data` as array or Null.